interface/reports/external_data.php

```diff
@@ -28,91 +28,12 @@
-                width: 32px;
-                height: 32px;
-                float: left;
-                margin:2px 6px 0 6px;
-                background-size: contain;
-            }
-            .dm-ed-in-4 {
-                display: none;
-            }
-            .dm-ed-in-5 {
-                margin-left: 10%;
-                margin-top: 3%;
-            }
-            .dm-ed-in-6 {
-                width: 20% !important;
-            }
-            .dm-ed-in-7 {
-                font-size: 12px;
-            }
-            .dm-ed-in-8 {
-                width: 60%;
-                margin-left: 10%;
-                margin-top: 40px;
-                border: 1px solid var(--gray400);
-                text-align: center;
-                padding: 30px;
-                font-size: 15px;
-                font-weight: bold;
-                background: var(--light);
-            }
-            .dm-ed-in-9 {
-                line-height: 30px;
-            }
-            ul.ext-tab-head li {
-                border-bottom: 2px solid var(--light);
-                color: var(--dark);
-                cursor: pointer;
-                display: inline-block;
-                font-size: 14px;
-                margin-bottom: 0;
-                padding: 3px 5px;
-                text-decoration: none;
-            }
-            ul.ext-tab-head li.child-active {
-                border-bottom: 2px solid #003366;
-            }
```

Comment:
Hi @tywrenn  and @bradymiller 

Although I remove the custome CSS style, but in `interface/themes/colors/utilities` still contains a `external-data.scss` file, maybe need you to check availability, if not I will remove it :)

Reply:
Yeah we'll have to see where it is used in other places

Reply:
I search the CSS class in all files, seems like didn't page use those class, so remove it.